sdks/go/pkg/beam/runners/universal/runnerlib/job.go

```diff
@@ -48,7 +48,7 @@ type JobOptions struct {
 
 // Prepare prepares a job to the given job service. It returns the preparation id
 // and artifact staging endpoint, if successful.
-func Prepare(ctx context.Context, client jobpb.JobServiceClient, p *pb.Pipeline, opt *JobOptions) (id, endpoint string, err error) {
+func Prepare(ctx context.Context, client jobpb.JobServiceClient, p *pb.Pipeline, opt *JobOptions) (id, endpoint, token string, err error) {
```

Comment:
stagingToken. Also please fix the comment above as well

Reply:
Done.